File: spring-data-requery/src/test/java/org/springframework/data/requery/repository/sample/DummyRepository.java
package org.springframework.data.requery.repository.sample;

import org.springframework.data.repository.CrudRepository;
import org.springframework.data.requery.domain.sample.Dummy;
import org.springframework.transaction.annotation.Transactional;

/**
 * DummyRepository
 *
 * @author debop@coupang.com
 * @since 18. 6. 25
 */
@Transactional
public interface DummyRepository extends CrudRepository<Dummy, Long> {

    // 모두 procedure 테스트용 예제이다. 
}

File: spring-data-requery-kotlin/src/test/kotlin/org/springframework/data/requery/kotlin/repository/sample/DummyRepository.kt
package org.springframework.data.requery.repository.sample

import org.springframework.data.repository.CrudRepository
import org.springframework.data.requery.kotlin.domain.sample.Dummy
import org.springframework.transaction.annotation.Transactional

/**
 * org.springframework.data.requery.repository.sample.DummyRepository
 *
 * @author debop
 */
@Transactional
interface DummyRepository: CrudRepository<Dummy, Int> {

    // 모두 procedure 테스트용 예제이다.

}